src/ui/ui-launch-button.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <button v-if="shouldDisplay" class="launch-ui-button" v-on:click="handleClick">
+    <span v-html="icon"></span>
+    <span>{{ spaceLogicCount }}</span>
+  </button>
+</template>
+
+<script>
+import spaceListIcon from '../../media/list-space.svg';
+import dom from '@nymag/dom';
+
+export default {
+  data() {
+    return {}
+  },
+  computed: {
+    icon() {
+      return spaceListIcon;
+    },
+    /**
+     * Grab the ref (data-uri) from the closest Space.
+     *
+     * @return {String}
+     */
+    spaceParentRef() {
+      const spaceEl = dom.closest(this.$store.state.ui.currentSelection.el, `[data-uri^="${this.$store.state.site.prefix}/components/clay-space"]`);
```

Comment:
withdrawn per discussion: we need the grandparent (the space) not the parent (the logic)